README.md

```diff
@@ -154,6 +199,42 @@ produce early errors because of recursive application of the first rule.
 - Let _targetName_ be ? Get(_target_, `"name"`).
 - If Type(_targetName_) is not `String`, then let _targetName_ be the empty string.
 - Let _status_ be ? SetFunctionName(_F_, _targetName_, `"bound"`).
+- Assert: _status_ is never used.
```

Comment:
The convention in the spec is to not have an assertion in those kinds of situations, though. (See CreateGlobalFunctionBinding, for instance)